.dev/entrypoint.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/ash
+cd /app
+
+tini -- php-fpm
+
+until nc -z -v -w30 $DB_HOST 3306
+do
+  echo "Waiting for database connection..."
+  # wait for 5 seconds before check again
+  sleep 5
+done
+
+#sed -i s/DB_HOST=127.0.0.1/DB_HOST=$DB_HOST/g /app/.env
+#sed -i s/DB_DATABASE=panel/DB_DATABASE=$DB_DATABASE/g /app/.env
+#sed -i s/DB_USERNAME=pterodactyl/DB_USERNAME=$DB_USERNAME/g /app/.env
+#sed -i s/DB_PASSWORD=/DB_PASSWORD=$DB_PASSWORD/g /app/.env
+
+#chown -R www-data:www-data .
+chmod -R 777 storage/* bootstrap/cache /var/run/php /app/.env
```

Comment:
This line should not be nessecary since the same is done during the image creation.

The only difference is `/app/.env`

Which if not already exisiting is first created in the next code section.